CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+Thank you for your interest in contributing! Join the [**Plugin Marketplace channel**](https://community.mattermost.com/core/channels/plugins-marketplace) on the Mattermost community server for discussion about this plugin.
+
+## Reporting Issues
+
+If you think you found a bug, [please use the GitHub issue tracker](https://github.com/mattermost/mattermost-marketplace/labels/issues/new) to open an issue.
+
+## Community Plugin
+
+To add your plugin to the Marketplace, please open [an issue using thus template](https://github.com/mattermost/mattermost-marketplace/issues/new?template=add_plugin.md). To update your plugin, please also open an issue [using this template](https://github.com/mattermost/mattermost-marketplace/issues/new?template=update_plugin.md).
+
+### Playbook
+
+This is a playbook used my Mattermost core committers.
+
+#### New plugin
+
+JIRA template:
+  - [PM](https://mattermost.atlassian.net/browse/MM-22224)
+  - [Dev](https://mattermost.atlassian.net/browse/MM-22221)
+  - [Security](https://mattermost.atlassian.net/browse/MM-22225)
+  - [QA](https://mattermost.atlassian.net/browse/MM-22223)
+
+After a new plugin has been submitted, the assignees of the issue posts the following message:
+```
+## Process checklist
+- [ ] Request review and create JIRA tickets and link them here:
+    - PM:
+    - Dev:
+    - Security:
+    - QA:
+- [ ] Fork repo under Mattermost oranisation
+- [ ] Cut plugin release
```

Comment:
Should we link to `cutPlugin` documentation? Should we be explicit that we only cut a release after review is completed? Do we tag our own releases explicitly (i.e. `mattermost-v2.1.1`)?

Reply:
> Should we link to `cutPlugin` documentation? 

Where can I find the documentation for `cutPlugin`?

> Should we be explicit that we only cut a release after review is completed? 

The wording on the dev doc make it quite clear that the fork and release cutting only take place after the reviews are done. Given that this is just a checklist, I don't think it doesn't have to be more clear.

> Do we tag our own releases explicitly (i.e. `mattermost-v2.1.1`)?

Why should this be needed? The tag are not visible to the public given that the repos are private.

Reply:
> Where can I find the documentation for cutPlugin?

I was thinking of https://developers.mattermost.com/internal/plugin-release-process/

> Given that this is just a checklist, I don't think it doesn't have to be more clear.

:+1:

> Why should this be needed? The tag are not visible to the public given that the repos are private.

I suppose it depends on invocation, but I usually see forks containing all the tags from upstream, and we'd want to explicitly disambiguate. Perhaps even require a signed commit from a core committer?

Reply:
> I suppose it depends on invocation, but I usually see forks containing all the tags from upstream, and we'd want to explicitly disambiguate. Perhaps even require a signed commit from a core committer?

:+1: for requiring signed commit in general.

My concern with a separate naming convention is that people might forget about it and we end up with inconsistencies. Maybe we defer this topic until the first plugin has been released and we can learn from the experience.